### DISCUSSION TITLE
先膜大佬，另外有打羽毛球的朋友吗，蹲伙伴一起做发球机

### DISCUSSION THREAD

**Discussion by NanshanTaotao** [peng-zhihui/ElectronBot#67] [General]
一个人做着太难了，我选的ESP32做主控，微信小程序来控制并编辑球路。
有没有单纯的爱好者，做好拿来练球哇，之前在b站上找到的都是团队想白嫖图纸、代码的。
目前我正在准备复试，能上岸的话还是很想把它做完的，开发自己喜欢的东西真的是一件很酷的事。

**Comment by adaelon** [comment]
我能加入吗，虽然是小白，但是想要积累一些项目经验，学习能力强，会做力所能及的事。

**Comment by iwestlin** [comment]
蹲一个捡球机
